apps/hashdotai/blog/039_self-building-database.mdx

```diff
@@ -0,0 +1,128 @@
+- Strongly typed entities, which support crosswalking via the type system we describe above.
+
+## Agentic AI
+
+To derive structure from the unstructured web we need intelligent agents. These agents need to be capable of running for an extremely long time, processing vast quantities of data. They need to be able to identify real patterns amidst noise, pick out the most important needles within unruly haystacks, and solve problems, perform calculations, and conduct analysis on the fly... introspecting and checking their own work, seeking confirmatory and contradictory sources, and formulating confidence assessments around the "synthesis" they produce.
+
+## A task executor
+
+Finally, to support complex agentic AI, we need a task executor capable of handling extremely long-running jobs. Thankfully, "durable execution" solutions such as _Temporal_ are now widely available, and open-source.
+
+# Solution
+
+**This isn't future technology.**
+
+In fact, [HASH](/) is already being used to solve each of the use cases outlined above.
+
+HASH is a new kind of database for integrating information, which grows, checks, and maintains itself, with a graphical user interface (GUI) that makes its contents directly accessible to all of its end-users. HASH provides all of the capabilities we "imagine" in our introduction.
+
+We founded _HASH, Inc._ as a research lab in 2019. In the years since, we've built the type system, graph, and application (in Rust, for speed, safety, and WASM-compatibility) steadily and quietly, testing it in the real-world throughout.
+
+If you'd like to use HASH yourself, we're now inviting users, and we invite you to [sign up](https://app.hash.ai/signup).
+
+- The [protocol](https://github.com/blockprotocol/blockprotocol) repository contains our **type system**, available under both the MIT and Apache 2.0 Licenses (dually, at your option).
+- Our [hash](https://github.com/hashintel/hash) repository which contains both:
+  - the **datastore**, licensed under AGPLv3 (like so many other great databases before it, prior to their shift away to proprietary licenses).
+  - the **application** which runs atop our datastore and makes the type system usable, while managing the hydration of the graph with information, is open-core. This means it is mostly AGPL-licensed, but partly proprietary (following a model virtually identical to that used by _GitLab_). You can read more about our [open-source philosophy](https://hash.dev/blog/open-source), which we've tried to be as transparent about as possible, as well as the other non-negotiable [commitments](https://hash.dev/blog/one-principle) we're making to our users.
+
+If you're a developer, we'd love for you to check out both.
```

Comment:
Maybe a link to `/contact` here for feedback / comments / questions / interest in work in the future?